src/tests/end-to-end/tests/target-page/issue-dialog.test.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { Browser } from '../../common/browser';
+import { launchBrowser } from '../../common/browser-factory';
+import { PopupPage } from '../../common/page-controllers/popup-page';
+import { TargetPage } from '../../common/page-controllers/target-page';
+import { scanForAccessibilityIssues } from '../../common/scan-for-accessibility-issues';
+
+describe('Target Page issue dialog', () => {
+    let browser: Browser;
+    let targetPage: TargetPage;
+    let popupPage: PopupPage;
+
+    beforeAll(async () => {
+        browser = await launchBrowser({ suppressFirstTimeDialog: true });
+        targetPage = await browser.newTargetPage();
+        popupPage = await browser.newPopupPage(targetPage);
+    });
+
+    afterAll(async () => {
+        if (browser) {
+            await browser.close();
+            browser = undefined;
+        }
+    });
+
+    it('should pass accessibility validation', async () => {
+        await popupPage.gotoAdhocPanel();
+        await popupPage.enableToggleByAriaLabel('Automated checks');
+
+        await targetPage.bringToFront();
+
+        const shadowRoot = await targetPage.getShadowRoot();
+        const issueHighlightLabel = await shadowRoot.$('.failure-label');
+        await issueHighlightLabel.click();
+
+        await targetPage.waitForSelector('.insights-dialog-main-container');
```

Comment:
We've been keeping the actual selector constants in a separate file so far; I'm kind of on the fence about whether that's still necessary now that we have the controller classes, but we should probably be consistent

Reply:
I'm debating if I should put the target page selectors in `common-selectors.ts` or create a new file `target-page-selectors.ts` for them. What are your thoughts?

Reply:
Went with `target-page-selectors.ts`